web/cobrands/fixmystreet/admin.js

```diff
@@ -123,5 +123,93 @@ $(function(){
             }
         }
     });
+
+    // Bits for the report extra fields form builder:
+
+    // Some elements' visibility depends on JS being available
+    $(".js-metadata-items .js-hidden").hide();
```

Comment:
We already have a `.hidden-js` class that will do this (and without a flash of content as it is in the CSS) :)